terms.html

```diff
@@ -93,10 +100,21 @@
   </dd>
   <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
   <dd>
-A role an <a>entity</a> can perform by asserting <a>claims</a> about one or
-more <a>subjects</a>, creating a <a>verifiable credential</a> from these
-<a>claims</a>, and transmitting the <a>verifiable credential</a> to a
-<a>holder</a>.
+A role an <a>party</a> can perform by <a>authoring</a> <a>claims</a>,
+constructing a <a>verifiable credential</a> from them,
+and transmitting that to a <a>holder</a>.
+  </dd>
+  <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
+  <dd>
+An <a>entity</a> that has 'a mind of its own', i.e.,
+it sets the objectives it aims to pursue,
+it autonomously gathers, stores, processes and disseminates data (including <a>identifiers</a>),
+it subjectively decides on the semantics of that data (i.e., what the data means),
+it authors <a>claims</a>, manages its own logic, makes its own decisions, e.g., regarding
+the kinds of <a>credentials</a> to <a>issue</a>, 
+what data in such <a>credentials</a> it considers to be <a>valid</a> for paricular purposes,
+etc.
```

Comment:
```suggestion
<ul>
  <li>it sets the objectives it aims to pursue</li>
  <li>it autonomously gathers, stores, processes, and disseminates data (including <a>identifiers</a>)</li>
  <li>it subjectively decides on the semantics of that data (i.e., what the data means)</li>
  <li>it <a>authors</a> <a>claims</a>, manages its own logic, and makes its own decisions regarding, for instance
  <ol>
    <li>the kinds of <a>credentials</a> to <a>issue</a></li>
    <li>the data in such <a>credentials</a> it considers to be <a>valid</a> for particular purposes</li>
  </ol>
  <li>etc.</li>
</ol>
```

Reply:
I've made some changes; pls check the text again.